android/src/main/groovy/com/redhat/ceylon/gradle/android/CeylonAndroidExtension.groovy

```diff
@@ -43,6 +43,16 @@ class CeylonAndroidExtension {
    */
   boolean skipJavaC
 
+  /**
+   * Absolute path to the ceylon executable.
+   */
+  def ceylonExecutable = "/usr/bin/ceylon"
```

Comment:
Cool! I meant to do this indeed. Can we have it default to `$CEYLON_HOME/bin/ceylon` IFF it's defined?